pkg/client/proxy_snapshot.go

```diff
@@ -46,26 +49,30 @@ func (c *ProxyClient) VolumeSnapshot(serviceAddress, volumeName string, labels m
 	}
 	recv, err := c.service.VolumeSnapshot(c.ctx, req)
 	if err != nil {
-		return "", errors.Wrapf(err, "failed to snapshot volume via proxy %v to %v", c.ServiceURL, serviceAddress)
+		log.Debug(err)
```

Comment:
Don't make the log here, but instead of letting the caller decide how to log.

If you are afraid of some info unable to disclosed in the returned err, then probably you need to enrich it w/ this info like the proxy server url.